services/IBF-pipeline/pipeline/lib/pipeline/rainfalldata.py

```diff
@@ -242,7 +240,7 @@ def findTrigger(self):
         # for leadtime in np.unique(df_thresholds.forecast_time.values):
 
         # threshold (1 degree)
-        df_leadtime = df_thresholds[df_thresholds.forecast_time == self.leadTimeValue]
+        df_leadtime = df_thresholds[df_thresholds['leadTime'] == self.leadTimeValue]
```

Comment:
@jannisvisser We should move these constant strings `leadTime` and `triggerLevel` to a separate constants file. Similar to how we have enums / models in TypeScript.

Reply:
I have done this now for the examples you commented on (when committed). In general, I agree with the sentiment also. But maybe good to know that the context here is:
- the IBF-pipeline has quite low coding standards compared to other parts of our code. There's A LOT that could/should be improved, so it feels a bit random to start with such a small insignificant detail, and leave all the rest. But I guess we have to start somewhere..
- also Python (at least the way we use it?) is not strongly typed in any way, which makes using enums less relevant right? Because it will not give you errors when you mistype or something.